api/namex/models/request.py

```diff
@@ -133,8 +133,7 @@ def has_consumed_name(self):
 
     @property
     def is_expired(self):
-        return self.stateCd == State.EXPIRED or \
-            (self.expirationDate and datetime.utcnow().date() > self.expirationDate.date())
+        return self.stateCd == State.EXPIRED
```

Comment:
Yes, as decided by Linda on Nov 24, that we would only look at the state.

The state is set by a daily job that we will now run right after 11:59 pm, so that NRs that expire at that time will have their new state without delay. (Currently there is a 7h delay between the expiry time and the state change, because the job is run at 7am.)